src/constructs/webcrawler/web-crawler-state-machine.ts

```diff
@@ -45,20 +120,44 @@ export default class WebCrawlerStateMachine extends Construct {
             // Continue crawling in another state machine execution
             continueExecution.next(new Succeed(this, 'ContinuingInAnotherExecution')))
             // Crawl every page we read from the queue in parallel
-            .otherwise(new Map(this, 'ForEachQueuedUrl', { itemsPath: '$.Payload.queuedPaths' })
-              .iterator(crawlPageAndQueueUrls).addCatch(readQueuedUrls)
-              .next(readQueuedUrls))
+            //.otherwise(new Map(this, 'ForEachQueuedUrl', { itemsPath: '$.Payload.queuedPaths' })
+            //  .iterator(crawlPageAndQueueUrls).addCatch(readQueuedUrls)
+            //  .next(readQueuedUrls))
+            // Change from Inline Map to Distributed Map
```

Comment:
Let's delete this too, users can look up the git history if they want to know what has changed :)

Reply:
Fixed